BUILD

```diff
@@ -43,13 +43,13 @@ launchctl load /Library/LaunchAgents/com.google.santagui.plist
 
 run_command(
     name = "reload",
-    srcs = ["//Source/santa-driver"],
+    srcs = ["//Source/santa_driver:santa-driver"],
```

Comment:
I'd suggest renaming the rule too, so you don't need to specify it.

Reply:
Done.